include/phasar/PhasarLLVM/Passes/GeneralStatisticsAnalysis.h

```diff
@@ -45,6 +45,10 @@ class GeneralStatistics {
   size_t LoadInstructions = 0;
   size_t MemIntrinsics = 0;
   size_t GlobalPointers = 0;
+  size_t Branches = 0;
+  size_t GetElementPtrs = 0;
+  size_t Terminators = 0;
```

Comment:
It seems to be redundant to count terminators as there is always exactly one terminator per basic-block

Reply:
good point, didn't think about that.